scripts/docker-build.py

```
@@ -2,120 +2,99 @@
 
 
 def build_image(
-    base_image: str, ros_distro: str, arch: str, tags: List[str], push: bool = False, env: ENV = {}
-):
-    print(f"\033[92mBuilding image with base image {base_image} and tags {tags}\033[0m")
-
+    arch: str,
+    tags: list[str],
+    push: bool = False,
+    no_cache: bool = False,
+    build_args: dict[str, str] = {},
```

Ruff (0.14.4)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)